queries/nix/injections.scm

```diff
@@ -29,7 +40,8 @@
     (indented_string_expression 
       ((string_fragment) @injection.content (#set! injection.language "bash")))
   ]
-  (#match? @_path "(^\\w+(Phase|Hook)|(pre|post)[A-Z]\\w+|script)$"))
+  (#match? @_path "(^\\w+(Phase|Hook)|(pre|post)[A-Z]\\w+|script)$")
+  (#set! injection.combined))
```

Comment:
The version before moving to upstream style doesn't have this. Can you briefly explain why it's needed?

Reply:
Consider me blind :+1: it's a good reason why.

Reply:
I'm back :smile:, sorry for not mentioning this. @dasJ can you add a comment above this to note why `injection.combined` is required? It'll be easier to track than reading commit message/PR text. After that I'll merge this PR. Doesn't cause much more performance issues from what I can notice (not that Nix files have good performance in the first place)